charts/kargo/templates/api/configmap.yaml

```
@@ -18,6 +18,9 @@ data:
   TLS_CERT_PATH: /etc/kargo/tls.crt
   TLS_KEY_PATH: /etc/kargo/tls.key
   {{- end }}
+  {{- if .Values.api.enableSecretManagement }}
```

```suggestion
  {{- if .Values.api.secretManagement.enabled }}
```

maybe? To match `.api.oidc.enabled`, .api.adminAccount.enabled`, etc.

---

I now notice there is a `.api.rollouts.integrationEnabled`, so I guess we have already lost this game.

I would prefer things to be uniform at some point, but will not let this block you now. So go for whatever you think is best.

---

Let's make an issue for improving the consistency pre-GA.